components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -1571,6 +1573,35 @@ func trackReadiness(ctx context.Context, gitpodService *gitpod.APIoverJSONRPC, c
 	}
 }
 
+func configureSSHDefaultDir() {
+	file, err := os.OpenFile("/home/gitpod/.bashrc", os.O_WRONLY|os.O_APPEND, 0o644)
+	if err != nil {
+		log.WithError(err).Error("cannot write .bashrc")
+	}
+	defer file.Close()
+	if _, err := file.WriteString("\nif [[ -n $SSH_CONNECTION ]]; then cd \"$GITPOD_REPO_ROOT\"; fi\n"); err != nil {
```

Comment:
Please don't use these env vars lightly. We should not hardcode them here and there.

Source of the truth in supervisor should be: https://github.com/gitpod-io/gitpod/blob/8d077b6eb5ea400b9a8be87d3a50494b335c6b2f/components/supervisor/pkg/supervisor/config.go#L195-L201